docs/install.md

```diff
@@ -20,94 +15,73 @@ store your Docker registry details in an environment variable. This
 install guide uses Google Container Registry (GCR) on GKE.
 
 ```
-export KF_REGISTRY=<your-container-registry>
-e.g: export KF_REGISTRY=gcr.io/<PROJECT_ID>
+export KF_REGISTRY=gcr.io/<PROJECT_ID>
 ```
 
 ## Install `kf` CLI
 
 The `kf` CLI is built nightly from the master branch. It can be downloaded
 from the following URLs:
 
-### linux
+### Linux
 > https://storage.googleapis.com/artifacts.kf-releases.appspot.com/nightly-builds/cli/kf-linux-latest
 ```sh
 wget https://storage.googleapis.com/artifacts.kf-releases.appspot.com/nightly-builds/cli/kf-linux-latest -O kf
 chmod +x kf
 sudo mv kf /usr/local/bin
 ```
 
-### mac
+### Mac
 > https://storage.googleapis.com/artifacts.kf-releases.appspot.com/nightly-builds/cli/kf-darwin-latest
 ```sh
 wget https://storage.googleapis.com/artifacts.kf-releases.appspot.com/nightly-builds/cli/kf-darwin-latest -O kf
 chmod +x kf
 sudo mv kf /usr/local/bin
 ```
 
-### windows
+### Windows
 > https://storage.googleapis.com/artifacts.kf-releases.appspot.com/nightly-builds/cli/kf-windows-latest.exe
 
-## Install dependencies
+## Create a Kubernetes cluster
 
-`kf` uses Istio to route HTTP requests to the running applications and Knative
-to deploy and scale applications.
+* Google Cloud: [Create a GKE cluster](/install/gke.md). Knative Serving and Istio will be installed with this cluster.
+* Existing: [Use an existing cluster](/docs/install/existing-cluster.md)
 
-> Note: Installing Istio and Knative Serve can be skipped if you are [using
-> Cloud Run on GKE](./install/Kf-with-CloudRun-on-GKE.md).
+## Install dependencies
 
-Install Istio:
+### Knative Build:
 
 ```.sh
-kubectl apply --filename https://raw.githubusercontent.com/knative/serving/v0.6.1/third_party/istio-1.1.3/istio-crds.yaml && \
-kubectl apply --filename https://raw.githubusercontent.com/knative/serving/v0.6.1/third_party/istio-1.1.3/istio.yaml && \
-kubectl label namespace default istio-injection=enabled
+kubectl apply --filename https://github.com/knative/build/releases/download/v0.6.0/build.yaml
 ```
 
-Install Knative Serve:
+> If you want more information about installing Knative, see [their docs][knative].
 
-```.sh
-kubectl apply --filename https://github.com/knative/serving/releases/download/v0.6.1/serving.yaml \
---filename https://github.com/knative/serving/releases/download/v0.6.1/monitoring.yaml \
---filename https://raw.githubusercontent.com/knative/serving/v0.6.1/third_party/config/build/clusterrole.yaml
-```
-
-Install Knative Build:
+### Service Catalog:
 
 ```.sh
-kubectl apply --filename https://github.com/knative/build/releases/download/v0.6.0/build.yaml
+kubectl apply -R -f third_party/service-catalog/manifests/catalog/templates
 ```
 
-> If you want to go more in depth installing Knative check out [their docs][knative].
-
 ## Install kf
 
-kf has controllers, reconcilers and webhooks that must be installed. The kf
+Kf has controllers, reconcilers and webhooks that must be installed. The kf
```

Comment:
It might be worth noting that if you install nightly you're getting a snapshot rather than a self-updating cluster.